ruby/rails6-mongo/app/Gemfile

```
@@ -0,0 +1,52 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.3'
+
+gem 'mongoid', '~> 7.0.5'
+gem "appsignal", :path => "/integration"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
+gem 'rails', '~> 6.0.4', '>= 6.0.4.4'
```

Can you use the latest Rails 6 version?

```suggestion
gem 'rails', '~> 6.1.4.4'
```